docs/en/cookbook/aggregate-fields.rst

```diff
@@ -175,12 +190,13 @@ relation with this method:
     <?php
     class Account
     {
-        public function addEntry($amount)
+        public function addEntry(int $amount): Entry
```

Comment:
That does look weird, not sure where it comes from.

Reply:
I don't see any usage of the return value so I removed it.